__tests__/index.test.js

```diff
@@ -154,3 +154,56 @@ describe('GET /api/articles/:article_id/comments', () => {
         });
     });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('PATCH:200 updates article votes by positive number', () => {
+        const voteChange = {inc_votes: 40}
+        return request(app)
+        .patch("/api/articles/1")
+        .send(voteChange)
+        .expect(200)
+        .then((response) => {
+            expect(response.body.article.votes).toBe(140)
+        })
+    });
```

Comment:
This is a good test, but to be more robust we can and should expect all of the properties in this article to be present as well.